pkg/apis/performance/v1/zz_generated.deepcopy.go

```diff
@@ -5,7 +5,7 @@
 package v1
 
 import (
-	v1 "github.com/openshift/custom-resource-status/conditions/v1"
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
```

Comment:
ok, this works and I don't really mind that much, but still, why exactly is this named import needed? what happens if you keep the default import?

Reply:
The file will be changed every time we run make generate. Better to do it ourselves, I think.